File: front/src/app/models/oeuvreartiste.ts
export class OeuvreArtiste {
    oeuvre: string;
    artiste: string;
    idMusee: string;

    constructor(oeuv: string, art: string, id: string) {
        this.oeuvre = oeuv;
        this.artiste = art;
        this.idMusee = id;
    }
    
}

File: MXPO-Back/src/main/java/co/simplon/MXPOBack/model/OeuvreArtiste.java
package co.simplon.MXPOBack.model;

import javax.persistence.Transient;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
@ToString
public class OeuvreArtiste {
	
	@Transient
	private String oeuvre;
	@Transient
	private String artiste;
	@Transient
	private String idMusee;
	
}
